src/main.cc

```diff
@@ -1587,7 +1592,7 @@ SquidMain(int argc, char **argv)
         Auth::Init();      /* required for config parsing. NOP if !USE_AUTH */
         Ip::ProbeTransport(); // determine IPv4 or IPv6 capabilities before parsing.
 
-        Format::Token::Init(); // XXX: temporary. Use a runners registry of pre-parse runners instead.
+        Format::Token::Init();
```

Comment:
FWIW, I am not against this change because I cannot understand what the removed XXX means, but if you want to keep this PR focused (after removing other comment changes), you may want to remove this change as well. Your call.

Reply:
Done in c6b6dbd